deepchem/hyper/gaussian_process.py

```diff
@@ -280,7 +285,8 @@ def optimizing_function(**placeholders):
       except NotImplementedError:
         pass
 
-      multitask_scores = model.evaluate(valid_dataset, [metric])
+      evaluator = Evaluator(model, valid_dataset, output_transformers)
+      multitask_scores = evaluator.compute_model_performance([metric])

```

Comment:
Why this change?  You could just pass the transformers to `evaluate()` and avoid having to create an Evaluator.

Reply:
I just copied `GridHyperparamOpt` implementation

https://github.com/deepchem/deepchem/blob/02fce35729826b1ef12a1cfa6519b491510217be/deepchem/hyper/grid_search.py#L156-L157

Reply:
What I mean is that all you need to do is add one argument to the existing call:

```python
multitask_scores = model.evaluate(valid_dataset, [metric], output_transformers)
```

Reply:
OK. I fixed.